docs/static/running-logstash-command-line.asciidoc

```diff
@@ -175,4 +175,3 @@ With this command, Logstash concatenates three config files, `/tmp/one`, `/tmp/t
 
 *`-h, --help`*::
   Print help
```

Comment:
Same comment as above. Do not trim trailing lines. It'll work most of the time, but if somebody adds two includes without a line break between them, your topic might blow up.